packages/bot-web-ui/src/stores/run-panel-store.js

```diff
@@ -313,12 +313,25 @@ export default class RunPanelStore {
     @action.bound
     showErrorMessage(data) {
         const { journal } = this.root_store;
+
         journal.onError(data);
         if (journal.journal_filters.some(filter => filter === message_types.ERROR)) {
             this.setActiveTabIndex(2);
+        } else {
+            this.root_store.ui.addNotificationMessage(journalError(this.switchToJournal));
         }
     }
 
+    @action.bound
+    switchToJournal = () => {
+        const { journal } = this.root_store;
```

Comment:
Can also get `ui` from the `root_store` and then reference it in lines 330-332.

```suggestion
        const { journal, ui } = this.root_store;
```